frontend/javascripts/admin/dataset/dataset_add_zarr_view.tsx

```diff
@@ -165,7 +176,13 @@ function DatasetAddZarrView(props: Props) {
               Add
             </AsyncButton>
           </FormItem>
-          <Divider />
+          <Collapse bordered={false} collapsible={exploreLog ? "header" : "disabled"}>
+            <Panel header="Exploration Log" key="1">
```

Comment:
```suggestion
            <Panel header="Log" key="1">
```

since the UI doesn't use the word "exploration" until now, I think